document-store/src/main/java/org/hypertrace/core/documentstore/query/Query.java

```diff
@@ -55,81 +50,241 @@
  *                 AggregateExpression.of(SUM, IdentifierExpression.of("col5")),
  *                 GTE,
  *                 ConstantExpression.of(100)))
- *         .sortingDefinition(
+ *         .sort(
```

Comment:
The comment section needs to be updated.

Reply:
Good catch.

Reply:
Done.